miio/miot_device.py

```diff
@@ -29,6 +29,16 @@ def get_properties_for_mapping(self) -> list:
         return self.get_properties(
             properties, property_getter="get_properties", max_properties=15
         )
+    
+    def get_property(self, property_key: str) -> dict:
+        property = {"did": property_key, **self.mapping.get(property_key)}
+
+        return self.get_properties(
+            [property], property_getter="get_properties", max_properties=1
+        )[0]
+
+    def get_value(self, property_key: str):
```

Comment:
Would it be better just to have only `get_property` returning directly the value? Please also add docstring(s).

Reply:
get_property returns the whole property including the `code` property so you can check the code for the status of the property
get_value returns the value directly

Reply:
Okay, fair enough. Could you improve the docstrings to contain example return values (that'll make it easy to see what to expect)?